web/src/__snapshots__/ShareSnapshotModal.test.tsx.snap

```diff
@@ -28,24 +28,59 @@ exports[`ShareSnapshotModal renders with modal open w/ known username 1`] = `
     style={Object {}}
     tabIndex="-1"
   >
+    <h2
+      className="ShareSnapshotModal-title"
+    >
+      Share a Shapshot
+    </h2>
     <div
       className="ShareSnapshotModal-pane"
     >
-      <a
-        href="http://test.com"
-        target="_blank"
+      <p>
+        Let anyone explore your current Tilt session with an interactive snapshot.
```

Comment:
I think it would be good to emphasize:
1) read-only
2) not live

to provide assurance that they're only sharing a limited thing (don't need to worry about anyone else modifying your stuff, don't need to worry about keeping your Tilt in the state you want the user to see, and don't need to worry about whether any secrets logged in the future are now available via the snapshot). Maybe some of that could be inferred from "snapshot", but more explicitness seems like it'd yield more feelings of safety.

stab: "Let anyone explore a read-only copy of the current state of your Tilt session with an interactive snapshot"

The "anyone" also kinda gives me pause, like "if I share this, can the whole WORLD see it!?"
We could say "anyone with the link", a la google docs. We do at least provide that level of privacy, and are not currently communicating it anywhere. That might be too wordy.

Copy isn't the point of this PR though. If you don't wanna mess with this, LMK and I'll file a ticket for us to deal with it later.

Reply:
This is great! This is precisely what I was thinking through as well, and trying to get a succinct version that sounds accessible and breezy is...tricky. I won't tinker further in this PR but I think we can have a story for "final round of copyediting" to address these branding and technical accuracy issues.